django/http/multipartparser.py

```diff
@@ -665,8 +700,8 @@ def __iter__(self):
 
 def parse_header(line):
     """ Parse the header into a key-value.
-        Input (line): bytes, output: unicode for key/name, bytes for value which
-        will be decoded later
+        Input (line): bytes, output: unicode for key/name,
```

Comment:
should wrap as close to 79 chars as possible. also convert to the more usual style:
```
"""
Parse ...
...
"""